autoscheduler/scraper/tests/api_tests.py

```diff
@@ -1,56 +1,61 @@
 from datetime import time
 from rest_framework.test import APITestCase, APIClient
 from scraper.models import Course, Department, Instructor, Meeting, Section
-from scraper.serializers import CourseSerializer, SectionSerializer, format_time
+from scraper.serializers import (CourseSerializer, SectionSerializer, TermSerializer,
+                                 CourseSearchSerializer, season_num_to_string,
+                                 campus_num_to_string, format_time)
 
 class APITests(APITestCase):
```

Comment:
```suggestion
class APITests(APITestCase): #pylint: disable=too-many-public-methods
```

Add this to silence this pylint error. We'll be refactoring this file in a later issue (#161), so it's not a problem right now.